maxim/logger/groq/utils.py

```
@@ -0,0 +1,222 @@
+                        {
+                            "index": 0,
+                            "message": {
+                                "role": "assistant",
+                                "content": str(completion),
+                            },
+                            "finish_reason": "stop",
+                        }
+                    ],
+                }
+
+        except Exception as e:
```

Ruff (0.11.9)

Do not catch blind exception: `Exception`

(BLE001)